platforms/jvm/language-java/src/main/java/org/gradle/api/tasks/JavaExec.java

```diff
@@ -525,6 +540,7 @@ public FileCollection getClasspath() {
      * {@inheritDoc}
      */
     @Override
+    @ToBeKeptEagerProperty(because = "Read-only nested property")
```

Comment:
❓ This inherits `@Nested`, right? So if we auto-ignore `@Nested`, this should not need an annotation either. Right?

Reply:
`@Nested` are now ignored